onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -296,11 +296,25 @@ def generate_identified_filename(filename: Path, identifier: str) -> Path:
     '''
     return filename.parent.joinpath(filename.stem + identifier).with_suffix(filename.suffix)
 
+def check_and_install_module(module):
+    import sys
+    import subprocess
+    import pkg_resources
+
+    required = {module}
+    installed = {pkg.key for pkg in pkg_resources.working_set}
+    missing = required - installed
+    if missing:
+        python = sys.executable
+        subprocess.check_call([python, '-m', 'pip', 'install', *missing], stdout=subprocess.DEVNULL)
+
 
 def write_calibration_table(calibration_cache):
     '''
     Helper function to write calibration table to files.   
     '''
+    check_and_install_module('flatbuffers')
```

Comment:
is it possible to output an error message if flatbuffers is not installed or install it if needed when ORT is installed? it seems a bit unexpected for additional dependencies to be installed from calling a function.

Reply:
I've made this module dependency to requirements.txt so that flatbuffers will be installed when ORT is installed.